### ISSUE TITLE
Issues with graph interpretation in Talos

### ISSUE THREAD

**Issue by franzalex1980** [autonomio/talos#262]
Thanks so much for coming here to raise an issue. Please take a moment to 'check' the below boxes:

- [ ] I'm up-to-date with the latest release:
    
      pip install -U talos

- [ ] I've confirmed that my Keras model works outside of Talos.

If you still have an error, please submit **complete trace** and a code with: 

- output of shape for x and y e.g. (212,12)
- Talos params dictionary
- The Keras model wired for Talos
- Description of extra variables in the model

You can provide the code in pastebin / gist or any other format you like. 

-------------------------------------------------------------------------
Hi, I have 3 questions on talos visualization please.

1. From your article, _hyperparameter optimization with keras 2018_, you stated that the hidden layers, learning rate and dropout do have a negative correlation with val_acc  (please see below the image):

**![grafik](https://user-images.githubusercontent.com/47783454/53890374-66424780-4028-11e9-962a-40be94559680.png)**

What does it mean for a negative correlation against the performance metric to occur under this condition? 

2. Secondly, the kernel density estimation, also from the same article as seen below:

**![grafik](https://user-images.githubusercontent.com/47783454/53894598-a73e5a00-4030-11e9-86a0-8bc315376b20.png)**

I understand that the x-axis represent parameter boundaries from the dropout dictionary. My question is are there existing values inbetween these generated boundaries in the graph with respect to the contour lines? Let me rephrase, since the countour lines are generated in that form, is it linked with values in the x-axis that are not visible?   

3. Lastly, out of curiosity, what type of algortihm those Talos operate on? 

Looking forward to your reply :) Thank you!

Ur Talos fan

Alex :)
